src/net.cpp

```diff
@@ -1855,8 +1855,8 @@ void CConnman::ThreadOpenConnections(const std::vector<std::string> connect)
                 std::tie(addr, addr_last_try) = addrman.Select();
             }
 
-            // Require outbound connections, other than feelers, to be to distinct network groups
-            if (!fFeeler && setConnected.count(m_netgroupman.GetGroup(addr))) {
+            // Require outbound IPv4/IPv6 connections, other than feelers, to be to distinct network groups
```

Comment:
`to be two`

https://github.com/jonatack/bitcoin/pull/1

Reply:
Since we are talking about direction (outbound) - either removing it or changing it to "toward" makes more sense - now that you mention it.

Note: Since the comment mentions "groups" it seems like a typo.